pcdsdevices/attenuator.py

```diff
@@ -311,21 +311,24 @@ def format_status_info(self, status_info):
 
         states = '\n'.join(render_ascii_att(blade_states))
 
-        energy = get_status_value(status_info, 'energy', 'value') * 1e3
-        energy_3rd = get_status_value(status_info, 'energy_3rd', 'value')
-        trans = get_status_value(status_info, 'position')
+        energy = get_status_float(
+            status_info, 'energy', 'value', scale=1e3, precision=3)
+        energy_3rd = get_status_float(
+            status_info, 'energy_3rd', 'value', scale=1e3, precision=3)
+        trans = get_status_float(
+            status_info, 'position', precision=4, format='E')
```

Comment:
Random thought, once we propagate variety information more thoroughly through the library then these sorts of calls can just check the display information from the variety metadata

Reply:
Very good point!